bin/hardening.sh

```diff
@@ -190,6 +190,23 @@ fi
 
 if [ "$BATCH_MODE" ]; then MACHINE_LOG_LEVEL=3; fi
 
+#
+# Math functions
+#
+
+function div() {
```

Comment:
Please put this function in `lib/common.sh`

Reply:
I consider this first but correct `lib/common.sh` sourcing depends correct `CIS_ROOT_DIR` (I have a test with a wrong CIS_ROOT_DIR) maybe in future you can consider add exit on error (`set -e`) on top script, and add in `hardening.sh` (entrypoint) self dependant function to have a correct behavior in all case. For moment, I fix with your change request.